File: 프로그래머스/unrated/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string myString, string pat) {
    int answer = 0;
    for(int i=0;i<myString.size();i++){
        string tmp = myString.substr(i, pat.size());
        if(pat == tmp) answer++;
    }
    return answer;
}

File: 프로그래머스/unrated/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java

class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        for(int i=0;i<myString.length() - pat.length()+1;i++){
            String sub = myString.substring(i, pat.length()+i);
            
            if(pat.equals(sub)){
                answer++;
            }
        }
        return answer;
    }
}